docs/docs/05-reusable-components.md

```diff
@@ -227,7 +227,31 @@ Counter.defaultProps = { initialCount: 0 };
 
 ### No Autobinding
 
-Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or arrow functions.
+Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or [arrow](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) `=>` functions.
+
+The example below illustrates `.bind(this)` being used to bind a method in the constructor.
+
+```javascript
+class Newsfeed extends React.Component {
+  constructor() {
+    super();
+    this.changeContent = this.changeContent.bind(this);
```

Comment:
I'm not really sure how I feel about having this as the canonical example, primarily because I don't like the idea that I'm defining the functions in one place and then mutating the functions later in my class in the constructor.

When I bind the function, I do it at the point that I'm passing the function to my component.  `<button onclick={changeContent.bind(this)}>Click Me</button>`.  Both styles have their pros and cons (mine is a little slower at runtime, because I'm not memoizing the bound function, but I'd gladly take the perf hit to avoid the mutation).